.travis.yml

```diff
@@ -41,18 +43,17 @@ matrix:
 
 
 before_install:
-  - ./ci-helpers/postgres_setup.sh
-  - wget http://bit.ly/miniconda -O miniconda.sh
-  - bash miniconda.sh -b -p $HOME/miniconda
-  - export PATH="$HOME/miniconda/bin:$PATH"
-  - conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
-  - conda update conda
-  - conda config --set show_channel_urls true
-  - conda config --add channels conda-forge --force
-  - conda create -n TEST python=$TRAVIS_PYTHON_VERSION --file requirements.txt --file requirements-dev.txt
-  - source activate TEST
-  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
-      conda install configparser ;
+  - if [[ $TEST_TARGET != 'docs' ]]; then
```

Comment:
We cannot build a python 3 env for `wofpy` yet b/c `odm2api` is not available, but `doctr` is a python 3 only library! So we need to have separate envs for testing and building the docs.